packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx

```diff
@@ -482,7 +499,7 @@ export const VideoForm: FC<VideoFormProps> = memo(({ onSubmit, setFormStatus })
         },
         disabled: videoFieldsLocked,
       }}
-      description="Royalties lets you earn commission from every sale of this NFT."
+      description="Disabling the comments section does not allow for posting new comments under this video and hides any existing comments made in the past."
```

Comment:
Why is this description used for royalties?

Reply:
lol, for some reason it was in the figma 😛

![Zrzut ekranu 2022-09-5 o 15 39 38](https://user-images.githubusercontent.com/13706246/188463011-270c9550-8360-4585-b342-25426ae2cd55.png)